tests/network-tests/src/flows/storage/initStorage.ts

```diff
@@ -53,6 +66,14 @@ export const singleBucketConfig: InitStorageConfig = {
   ],
 }
 
+export const fiveBucketConfig: InitStorageConfig = {
```

Comment:
This should not be needed after https://github.com/Joystream/joystream/pull/4007

Reply:
Adressed in [6003fb6](https://github.com/Joystream/joystream/pull/3997/commits/6003fb6acb0b6a9cb941e82b79bde7eb8267b95e)